gateway/gateway-demo.yaml

```diff
@@ -4,11 +4,15 @@ metadata:
   name: gateway-demo
 spec:
   api:
-    serverUrl: https://test-gateway.spring.animalrescue.online
+serverUrl: 
+    serverUrl: http://animal-rescue.my.domain.io
     title: "Animal Rescue"
     description: "Sample application for Spring Cloud Gateway commercial product demos."
     documentation: "https://github.com/spring-cloud-services-samples/animal-rescue/"
     version: "1.0.0-K8s"
+    cors:
+      allowedOrigins:
+        - "*"
```

Comment:
Does this actually work? I thought a customer reported that Spring Security no longer support `*` as origin. Maybe it's better to write an arbitrary api portal url here and document that they should match.

Reply:
Yes, this does work. I think that they had other typo in their configuration file from what little I remember. I will modify this though to a fake api-portal domain as that makes more sense.